File: Desktop/SummerVacation/0701/src/numTwentyone.java
import java.util.Scanner;
public class numTwentyone {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        String mon[] = {"January","February","March","April","May","June","July","August","September","October","November","December"};
        System.out.print("What is the number of the month? ");
        int number = scan.nextInt();
        System.out.printf("%d is %s.",number,mon[number - 1]);
    }
}

File: Desktop/SummerVacation/0701/python/numTwentyone.py
def switch(x):
    return{
        1 : "January",2 : "February", 3 : "March", 4 : "April", 5 : "May", 6 : "June", 7 : "July", 8 : "August",
        9 : "September", 10 : "October", 11 : "November", 12 : "December"
        }.get(x,"Wrong")
num = input("Enter the number of the month: ")
print("The name of the month is",switch(int(num)))